src/release_artifacts_resources/ios/cdk/cdk/credential_rotation/lambda_functions/credential_rotator.py

```diff
@@ -145,30 +185,27 @@ def update_env_vars(
     temporary_credentials: Dict[str, str],
     token: str,
     project_path: str,
-    *,
-    max_attempts=MAX_RETRY_ATTEMPTS,
-    max_wait=MAX_RETRY_WAIT,
-    log=True,
 ) -> None:
-    url = CIRCLECI_URL_TEMPLATE.format(project_path=project_path)
-    headers = {"Circle-Token": token}
 
-    @retry(max_attempts=max_attempts, max_wait=max_wait, log=log)
-    def update(env_var_name: str, env_var_value: str) -> None:
-        payload = {"name": env_var_name, "value": env_var_value}
-        response = requests.post(url, json=payload, headers=headers)
-        if not is_successful_response(response):
-            safe_content = response.text.replace(env_var_value, "*" * len(env_var_value))
+    for key, value in temporary_credentials.items():
+        update(key, value, token, project_path)
 
-            raise RuntimeError(
-                "Could not update env var "
-                + f"key={env_var_name} "
-                + f"status_code={response.status_code} "
-                + f"body={safe_content}"
-            )
 
-    for key, value in temporary_credentials.items():
-        update(key, value)
+@retry()
+def update(env_var_name: str, env_var_value: str, token: str, project_path):
```

Comment:
nit: add type hint to `project_path`
```suggestion
def update(env_var_name: str, env_var_value: str, token: str, project_path: str):
```

Reply:
Updated